exp-player/addon/components/exp-exit-survey/component.js

```diff
@@ -121,23 +122,44 @@ export default ExpFrameBaseComponent.extend(Validations, FullScreen, {
             this.send('next');
         }
     },
-    currentSessionsCompleted: Ember.computed('frameContext', function() {
-        var pastSessions = this.get('frameContext.pastSessions');
-        if (pastSessions) {
-            return pastSessions.get('length') || 1;
+    currentSessionStatus: Ember.computed('frameContext', function() {
+        // Count all sessions
+        //var pastSessions = this.get('frameContext.pastSessions');
+        //if (pastSessions) {
+        //    return pastSessions.get('length') || 1;
+        //}
+
+        // Count only sessions with at least minVideos pref-phys-videos frames
+        var expData = this.get('frameContext.pastSessions').map((session) => session.get('expData'));
+        var expDates = this.get('frameContext.pastSessions').map((session) => moment(session.get('createdOn')));
+        var nSessions = 0;
+        var sessionDates = [];
+        for (var iSess = 0; iSess < expData.length; iSess++) {
+            var keys = Object.keys(expData[iSess]);
+            var nVideos = 0;
+            for (let i = 0; i < keys.length; i++) {
+                var frameKeyName = keys[i];
+                if (frameKeyName.indexOf('pref-phys-videos') !== -1) {
+                    nVideos++;
+                }
+            }
+            if (nVideos >= this.get('minVideosToCountSession')) {
+                nSessions++;
+                sessionDates.push(expDates[iSess]);
+            }
         }
-        return 1;
+        var minDate = moment.min(sessionDates);
+        var maxDate = moment.max(sessionDates);
+        return [nSessions, maxDate.diff(minDate, 'days') + 1];
```

Comment:
Not a fan of returning arrays, and later accessing the first or second item in the array. The context of what you are doing gets lost.    I would return a dictionary here:
```
        return {'nSessions': nSessions, 'daysSessionsCompleted': moment.max(sessionDates).diff(moment.min(sessionDates), 'days') + 1 };
```